.storybook/main.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
       name: '@storybook/preset-scss',
       options: {
         cssLoaderOptions: {
-          modules: true,
+          modules: {
+            auto: true,
+          },
```

Comment:
Required tweak to work with nextJS

Reply:
Consider inline comment. ✍️

Reply:
I mean, this whole config's purpose is 'work with NextJS'?

Reply:
That's fair. 👯